.github/workflows/nightly-it-example-generator.yml

```diff
@@ -25,11 +25,32 @@ env:
   MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dcheckstyle.skip=true -Dspotless.apply.skip=true -Dmaven.javadoc.skip=true
 
 jobs:
+  build-cache:
+    if: github.repository == 'apache/shardingsphere'
+    name: Build Cache
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: shardingsphere-maven-third-party-cache-${{ github.sha }}
+          restore-keys: |
+            shardingsphere-maven-third-party-cache-
+            shardingsphere-maven-third-party-
+      - name: Build with Maven
+        run: ./mvnw -B -T1C -ntp clean install -DskipITs -DskipTests
   example-generator:
```

Comment:
how about rename job name as `generate-and-test-examples`

Reply:
done.